docs/Parameters.md

```diff
@@ -54,8 +54,8 @@ The parameter format is `key1=value1 key2=value2 ... ` . And parameters can be s
   * support multi validation data, separate by `,`
 * `num_iterations`, default=`100`, type=int, alias=`num_iteration`,`num_tree`,`num_trees`,`num_round`,`num_rounds`
   * number of boosting iterations
-  * note: `num_tree` here equal with `num_iterations`. For multi-class, it actually learns `num_class * num_iterations` trees.
-  * note: For python/R package, cannot use this parameters to control number of iterations.
+  * note: For python/R package, **this parameters is ignored**.
```

Comment:
Yes, but they are not equal in meanings. The parameter is not ignored: it was renamed in R/Python as a mandatory argument of the training function with a default value (hence why specifying the parameter explicitly does not apply ("cannot use this parameters to control number of iterations."), as it is already overridden).

See example here: https://github.com/Microsoft/LightGBM/blob/master/R-package/R/lgb.train.R#L220